### PR TITLE
Drop telepathy-gabble-xmpp-console

### PR DESCRIPTION
**Pull request by rainemak**
There are two main reasons
1) xmpp-console pulls python3 to the default image and
2) xmpp-console uses Gtk python bindings

In addition, this PR contains two other commits

- Change mktests.sh to git formatted patch
- Git format worky-fix-mem-leak patch

With this it's nicer & simpler to build this locally.

### CODE REVIEW

File: rpm/telepathy-gabble.spec [hunk 1]
```diff
@@ -107,14 +103,15 @@ sh tests/mktests.sh > %{buildroot}/opt/tests/%{name}/tests.xml
 install -m 0644 tests/INSIGNIFICANT %{buildroot}/opt/tests/%{name}/INSIGNIFICANT
 install -m 0644 tests/README %{buildroot}/opt/tests/%{name}/README
 
+rm -f %{buildroot}/%{_bindir}/%{name}-xmpp-console
```
Comment (pvuorela) [inline rpm/telepathy-gabble.spec]: Often the ignored content is marked with %exclude on %files. Could be nicer here?

File: rpm/telepathy-gabble.spec [hunk 2]
```diff
@@ -5,15 +5,14 @@ Release:    1
 License:    LGPLv2+
 URL:        https://telepathy.freedesktop.org/
 Source0:    %{name}-%{version}.tar.bz2
-Source1:    mktests.sh
-Source2:    INSIGNIFICANT
```
Comment (pvuorela) [inline rpm/telepathy-gabble.spec:9]: Delete these too.

Comment (rainemak) [reply rpm/telepathy-gabble.spec:9]: These files are still installed and part of the tests package but coming from the patch file.

Comment (pvuorela) [reply rpm/telepathy-gabble.spec:9]: Sure, but the files under rpm/ should be gone. I don't think you get past gating with them.

File: rpm/telepathy-gabble.spec (hunk 1 above)
Comment (rainemak) [reply rpm/telepathy-gabble.spec]: Maybe that's nicer indeed.

File: rpm/telepathy-gabble.spec (hunk 2 above)
Comment (rainemak) [reply rpm/telepathy-gabble.spec:9]: Sorry, through that I nuked them long time back (like two weeks ago). Now gone.

**Comment by rainemak** [comment]
Also added ownership for the /opt/tests/telepathy-gabble

**Review by pvuorela — APPROVED** [review]

**Pull request merged**